Bot.rb

```diff
@@ -0,0 +1,103 @@
+require 'telegram/bot'
+require 'dotenv'
+require 'sqlite3'
+Dotenv.load
+$token = ENV["BOT_TOKEN"]
+step_1 = {
+  story: " -Смотри! Голуби как высоко-то летают...\n Сказать что это не голуби?",
+  answers: ["Да", "Хорошо летят же, зачем тревожить"],
+  right_answer: "Да",
+  fail: "Дракончики догнали тебя и немножко съели =("
+}
+step_2 = {
+  story: "Вы приземлились на полянке. \n-Ух! Еле отвязались от них. Кушать будешь?",
+  answers: ["Да", "Нет"],
+  right_answer: "Да",
+  fail: "Брезгуешь значит? Шахалай махалай зверем будешь так и знай!\n\nТеперь ты зверушка"
+}
+step_3 = {
+  story: "Ну, вот и перекусили. Чего нужно-то от меня?",
+  answers: ["Сказку", "Самогону конечно же!"],
+  right_answer: "Сказку",
+  fail: ".........\n\nПрошул день\n-Wakeup Neo!"
+}
+step_4 = {
+  story: "Сказка будет, когда бд прицепит 1 Неуч!!!!",
+  answers: ["Эх", "Чего это он неуч? Он почти молодец ))))"],
+  right_answer: "Эх",
+  fail: "Не все считают так же! Скалкой в плечи и ты полетел"
+}
+$steps = [step_1, step_2, step_3, step_4]
+HELLO =  <<-HELLOSTRING
+Hello! I'm pretty young story bot
+Usage:
+      /story    - for start little story jorney
+      /help     - this text
+      /hi       - and i will say hello to you :)
+HELLOSTRING
+
+current_step =-1
+$last_step = 3
+$status = 0
+check = false
```

Comment:
those 4 variables (lines 39-42) are good candidates to be instantiated in a class and be defined in initializer